openedx/features/genplus_features/genplus_learning/signals.py

```diff
@@ -147,3 +133,8 @@ def program_deleted(sender, instance, **kwargs):
 def program_updated(sender, instance, created, **kwargs):
     if not created:
         ProgramCache.clear_mapping_for_all_courses(instance)
+@receiver(SignalHandler.course_published)
+def course_published(course_key, **kwargs):
+    print(kwargs)
```

Comment:
remove print statements. what we need this `course_published` handler?

Reply:
I removed it already